### PR TITLE
Create reviewer-checklist.md

### PR DESCRIPTION
**Pull request by vantuyls**
@tobyhodges I'm assuming this can just live at the root directory, as I don't see a way to include this as a template when responding to an issue

Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

---

### CODE REVIEW

File: reviewer-checklist.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
+<H2>Reviewers, please use the following checklists as guidance for your review:</H2>
```
Comment (tobyhodges) [inline reviewer-checklist.md]:
```suggestion
# Review Checklist

Reviewers, please use the following checklists as guidance for your review:
```

File: reviewer-checklist.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+<H2>Overall Impression of the Submission</H2>
```
Comment (tobyhodges) [inline reviewer-checklist.md]:
```suggestion
## Overall Impression of the Submission
```

File: reviewer-checklist.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Please give the submitter and other reviewers your overall impression of the dataset and supplementary materials
+
+
+
+<H2>Data Organisation</H2>
```
Comment (tobyhodges) [inline reviewer-checklist.md]:
```suggestion
## Data Organisation
```

File: reviewer-checklist.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+The dataset is organized according to the following principles:
+
+
+- [ ] The data is "Tidy" (if appropriate to the data type)
```
Comment (tobyhodges) [inline reviewer-checklist.md]: Perhaps we can include a link to a resource for the reviewer to learn more about what this means. I think we have one such link already in the README?

File: reviewer-checklist.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+- [ ] The data is human readable
+- [ ] The data is presented in a plain text or appropriate database format
+- [ ] The dataset is not too small or unnecessarily large for the intended lesson type
+
+
+
+<H2>Documentation</H2>
```
Comment (tobyhodges) [inline reviewer-checklist.md]:
```suggestion
## Documentation
```

File: reviewer-checklist.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+- [ ] Use cases
+- [ ] Keywords
+- [ ] File/Variable Descriptions
+- [ ] Codes
+- [ ] Hazards
+- [ ] Processing
+- [ ] Citation
+
+
+<H2>License</H2>
```
Comment (tobyhodges) [inline reviewer-checklist.md]:
```suggestion
## License
```

File: reviewer-checklist.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+To eliminate barriers to reuse, datasets in the collection should be released into the public domain, i.e. published with the Creative Commons Zero waiver (CC0). We will also accept datasets published with the Creative Commons Attribution 4.0 International license (CC-BY), but we encourage submitters to consider that CC-BY is not considered appropriate for licensing data and the license may have the unintended effect of making it more difficult for others to reuse the data.
+
+
+- [ ] The dataset has been assigned CC-0 license in a data.txt file
+- [ ] The documentation for the dataset has been assigned a CC-BY license in a documentation.txt file
+- [ ] Any code or software provided with the dataset has been assigned an MIT license in a code.txt file
+- [ ] The license is clearly labelled and in a LICENSES folder
+
+<H2>Supplementary Materials and Information</H2>
```
Comment (tobyhodges) [inline reviewer-checklist.md]:
```suggestion
## Supplementary Materials and Information
```

File: reviewer-checklist.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+Code:
+- [ ] Code is sufficiently internally documented
+- [ ] Code is written in a preferred language (R, Python)
+- [ ] Code is presented in a non-preferred language for reasons that are well described
+- [ ] References are included for related or "upstream" software and the references (e.g. URLs, DOIs) are functional
+
+Teaching Materials:
+- [ ] Teaching materials are presented in a generalized way that allows people unfamiliar wtih the data and/or material to teach it with relatively little preparation time
+- [ ] Teaching materials are presented in an open format (e.g. plain text, pdf)
```
Comment (tobyhodges) [inline reviewer-checklist.md]:
```suggestion
- [ ] Teaching materials are presented in an open format (e.g. plain text, pdf, R Markdown, Jupyter Notebook)
```

**Review by tobyhodges — COMMENTED** [review]
Looks good. I've suggested a few changes to switch from HTML heading syntax into pure Markdown.

**Comment by vantuyls** [comment]
okay, i've accepted changes and updated the link. as soon as this is merged, i'll send off the email to the working group. i haven't had time this week (so far) to get to the GH Pages site, but still plan on it.

**Review by tobyhodges — APPROVED** [review]

**Comment by tobyhodges** [comment]
It looks like you clicked "mark conversation as resolved" instead of committing the suggested changes, so I have made those edits directly now and can merge this.

**Pull request merged**